packages/blitz/package.json

```diff
@@ -32,7 +32,7 @@
     "express": "4.17.3",
     "http": "0.0.1-security",
     "node-fetch": "3.2.3",
-    "npm-which": "3.0.1",
+    "npm-which": "^3.0.1",
```

Comment:
Maybe I'm a bit paranoid after `faker.js` and `node-ipc`, but wouldn't it be better to use the absolute version ( w/o `^` )?

Reply:
Yeah, good point